linkerd/app/src/env.rs

```diff
@@ -82,6 +82,9 @@ pub const ENV_BUFFER_CAPACITY: &str = "LINKERD2_PROXY_BUFFER_CAPACITY";
 pub const ENV_INBOUND_ROUTER_MAX_IDLE_AGE: &str = "LINKERD2_PROXY_INBOUND_ROUTER_MAX_IDLE_AGE";
 pub const ENV_OUTBOUND_ROUTER_MAX_IDLE_AGE: &str = "LINKERD2_PROXY_OUTBOUND_ROUTER_MAX_IDLE_AGE";
 
+const ENV_INBOUND_MAX_IDLE_PER_ENDPOINT: &str = "LINKERD2_PROXY_MAX_IDLE_PER_ENDPOINT";
```

Comment:
not/tioli: should this maybe be "MAX_IDLE_CONNS" or something? we also use the string "MAX_IDLE" to refer to the service cache eviction timeout, which is a pretty different thing...